web/webapp/testSetupDebug.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env sh
+set -eux
+if [ ! -f ".env" ]
+then
+    cp .env.local.example .env
+fi
+
+mkdir -p .logs
+
+
+
+echo "starting redis server"
+redis-server > .logs/redis-logs.txt &
+PID_redis="${!}"
+sleep 15
+# trap 'if ps -p ${PID_redis} > /dev/null; then kill ${PID_redis}; fi' EXIT
```

Comment:
Why are those traps commented? They look good and it seems like without them, Redis is gonna run even after executing this script.

Reply:
you are right we can leave the traps uncommented and it will still stay open with the launched ui for manual debugging until it's closed.

Reply:
changed it